deno/fhirchat-demo/routes/api/aiConversation.ts

```diff
@@ -26,11 +26,11 @@ function handler(req: Request): Response {
       assistant = await createAssistantAgent();
       assistant.events.on(
         MODEL_OUTPUT_EVENT,
-        (message, agentName, toolName) => {
+        (message: string, agentName: string, toolName: string) => {
```

Comment:
ah this will clash with my changes for the thought process. Good to have the typing!